.github/workflows/test.yml

```
@@ -104,151 +104,147 @@ jobs:
         run: cabal build
 
       - name: Set test options
-        # run the tests without parallelism, otherwise tasty will attempt to run
-        # all functional test cases simultaneously which causes way too many hls
-        # instances to be spun up for the poor github actions runner to handle
-        #
         # See https://github.com/ocharles/tasty-rerun/issues/22 for why we need
         # to include 'new' in the filters, since many of our test suites are in the
         # same package.
         run: |
-          echo "TEST_OPTS=-j1 --rerun-update --rerun-filter failures,exceptions,new" >> $GITHUB_ENV
+          echo 'cabal configure --test-options="--rerun-update --rerun-filter failures,exceptions,new"' >> cabal.project.local
```

I mean this, which has the side effect of writing those options into cabal.project.local
```suggestion
          cabal configure --test-options="--rerun-update --rerun-filter failures,exceptions,new"
```

```
$ cabal configure --test-options="--rerun-update --rerun-filter failures,exceptions,new"
$ cat cabal.project.local 
ignore-project: False
test-options: "--rerun-update"
              "--rerun-filter"
              "failures,exceptions,new"
```

---

That's what I meant to type, just :facepalm: